src/components/toast/toast.stories.mdx

```diff
@@ -0,0 +1,114 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import styled from 'styled-components';
+import Toast from '.';
+import { useState, useEffect } from 'react';
+import { select, text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import OptionsHelper from '../../utils/helpers/options-helper/options-helper';
+import Button from '../button';
+
+<Meta title="Design System/Toast" />
+
+# Toast
+
+```javascript
+import Toast from 'carbon-react/lib/components/toast';
+```
+### Grouped (stacking):
+
+<Preview>
+  <Story name="stacked" parameters={{ info: { disable: true }}} >
+  {() => {
+    const variant = select('variant', OptionsHelper.toast, OptionsHelper.toast[0]);
+    const variantB = select('variant', OptionsHelper.toast, OptionsHelper.toast[1]);
+    const children = text('children', 'My message');
+    const id = text('id', 'default-toast');
+    const idB = text('id', 'default-toast-b');
+    const onDismiss = boolean('onDismiss', true);
+    const isCenter = boolean('isCenter', true);
+    const onDismissClick = onDismiss ? (evt) => { action('click')(evt); } : undefined;
```

Comment:
Might be worth setting `isOpen` to false when the close Icon is clicked so the Toast closes etc

Reply:
Due to me implementing it through button toggle I wanted to avoid Toast Close Icon click(closing) so that isOpen state is controlled through storybook  and button toggle.

Reply:
How are we planning on testing the functionality when a single toast in a `stack` is closed/dismissed, I think there's value in displaying that behaviour in a storybook demo as currently all that happens is the the stacked toasts appear and dismiss at the same time

Reply:
updated.